test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -4,12 +4,19 @@
 
 import org.junit.Test;
 
+import seedu.addressbook.data.exception.IllegalValueException;
+
 public class NameTest {
 
     @Test
     public void isSimilar_similarNames() {
-        final Name inputName = new Name("John Smith");
-        final Name otherName = new Name("John Smith");
-        assertTrue(inputName.isSimilar(otherName));
+        try {
+            final Name inputName = new Name("John Smith");
+            final Name otherName = new Name("John Smith");
+            assertTrue(inputName.isSimilar(otherName));
+        } catch (IllegalValueException e) {
```

Comment:
Instead of handling exceptions inside the test, you can set the test method to throw the exception.